packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -649,7 +645,18 @@ async function loadRemoteParameterOptions() {
 			credentials: node.value.credentials,
 		});
 
-		remoteParameterOptions.value = remoteParameterOptions.value.concat(options);
+		const [safeOptions, invalidOptions] = partition(options, isINodePropertyOptions);
```

Comment:
`isINodePropertyOptions` doesn't seem to check if `options.value` is `undefined`, only that it has a `value` property so it might not catch this

```ts
export const isINodePropertyOptions = (
	item: INodePropertyOptions | INodeProperties | INodePropertyCollection,
): item is INodePropertyOptions => 'value' in item && 'name' in item && !('displayName' in item);
```

Reply:
Also, do we know that the issue is coming from the remote options and not passed in as props? If it's coming as props, we don't know catch the issue, only filter it out

Reply:
I think it's not possible because our types would prevent you from defining options without `value`. But let's be safe, maybe some nodes use type casts or expressions.